ui/page/collection/internal/collectionPublish/view.jsx

```diff
@@ -308,16 +293,16 @@ function CollectionForm(props: Props) {
         </TabPanels>
       </Tabs>
 
-      {!loading && (
+      {activeChannelClaim !== undefined && (
         <Card
           className="card--after-tabs"
           actions={
             <>
               <div className="section__actions">
                 <Button
                   button="primary"
-                  disabled={isBuiltin || creatingCollection || updatingCollection || Boolean(submitError) || !hasClaims}
-                  label={creatingCollection || updatingCollection ? __('Submitting') : __('Submit')}
+                  disabled={isBuiltin || isPublishing || Boolean(submitError) || !hasClaims}
```

Comment:
This looks nice.

:small_blue_diamond: Opinion:  maybe don't need the elipsis, then have the same padding as the right (`pad-text-pad-spinner-pad`)?  The ellipsis seems redundant with the spinner, plus we don't need to retranslate the string.  But good enough as is.